src/binary-encoding.adoc

```diff
@@ -758,27 +754,37 @@ consume only enough bits to represent the fractional precision supported by the
 | Unused
 |===
 
+NOTE: Timestamps are encoded in Little Endian byte order. In the diagrams below, the bytes are read from left to right
+(least significant to most significant), while the bits within each byte should be read from right to left. (Again
+least significant to most significant.) +
+ +
+While this encoding may complicate human reading, it guarantees that the timestamp's subfields (`year`, `month`,
+etc.) occupy the same bit indexes regardless of how many bytes there are overall. (The last subfield,
+`fractional_seconds`, always begins at the same bit index when present, but can vary in length according to the
+precision.) This allows processors to read the Little-Endian bytes into an integer and then mask the appropriate
+bit ranges to access the subfields.
+
 ==== Opcode `0x70`: Year (1 byte)
 [source]
 ----
 +=========+
-|YYYY:YYY-|
+|-YYY:YYYY|
 +=========+
 ----
 
 ==== Opcode `0x71`: Month (2 bytes)
 [source]
 ----
 +=========+=========+
-|YYYY:YYYM|MMM-:----|
+|MYYY:YYYY|----:-MMM|
```

Comment:
Yeah this is what I was talking about elsewhere, this feels more consistent endian-ness.